frontend/src/components/OdhDocCard.tsx

```diff
@@ -138,22 +136,24 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFavorit
       isSelected={selected}
       isSelectable
     >
-      <CardHeader>
-        <CardHeaderMain style={{ maxWidth: '33%', width: '100%' }}>
-          <BrandImage
-            src={odhDoc.spec.img || odhDoc.spec.icon || ''}
-            alt={odhDoc.spec.displayName}
-          />
-        </CardHeaderMain>
-        <CardActions hasNoOffset>
-          <FavoriteButton isFavorite={favorite} onClick={() => updateFavorite(!favorite)} />
-        </CardActions>
+      <CardHeader
+        actions={{
+          actions: (
+            <>
+              <FavoriteButton isFavorite={favorite} onClick={() => updateFavorite(!favorite)} />
+            </>
```

Comment:
This appears to be a missed point of my previous review... mind removing the fragment here too?